bundles/org.openhab.binding.gree/README.md

```diff
@@ -0,0 +1,147 @@
+|               |           | Mode can be one of auto/cool/eco/dry/fan/heat or on/off                                     |
+|               |           | Check the Air Conditioner's operating manual for supported modes.                                 |
+| temperature   | Number    | Sets the desired room temperature                                                                 |
+| air           | Switch    | Set on/off the Air Conditioner's Air function if applicable to the Air Conditioner model          |
+| dry           | Switch    | Set on/off the Air Conditioner's Dry function if applicable to the Air Conditioner model          |
+| health        | Switch    | Set on/off the Air Conditioner's Health function if applicable to the Air Conditioner model       |
+| turbo         | Switch    | Set on/off the Air Conditioner's Turbo Mode.                                                      |
+| quiet         | String    | Set Quiet Mode: off/auto/quiet                                                                    |
+| swingUpDown   | Number    | Sets the vertical (up..down) swing action on the Air Conditioner,                                 |
+|               |           | OFF: 0, Full Swing: 1, Up: 2, MidUp: 3, Mid: 4, Mid Down: 5, Down : 6                             |
+| swingLeftRight| Number    | Sets the horizontal (left..right) swing action on the Air Conditioner                             |
+|               |           | OFF: 0, Full Swing: 1, Left: 2, Mid Left: 3, Mid: 4, Mid Right: 5, Right : 6                      |
+| windspeed     | Number    | Sets the fan speed on the Air conditioner Auto:0, Low:1, MidLow:2, Mid:3, MidHigh:4, High:5       |
+|               |           | The number of speeds depends on the Air Conditioner model.                                        |
+| powersave     | Switch    | Set on/off the Air Conditioner's Power Saving function if applicable to the Air Conditioner model |
+| light         | Switch    | Enable/disable the front display on the Air Conditioner if applicable to the Air Conditioner model|
+|               |           | Full Swing: 1, Up: 2, MidUp: 3, Mid: 4, Mid Down: 5, Down : 6                                     |
+
+
+When changing mode, the air conditioner will be turned on unless "off" is selected.
+
+## Full Example
+
+**Things**
+
+```
+Thing gree:airconditioner:a1234561 [ ipAddress="192.168.1.111", refresh=2 ]
+```
+
+**Items**
+
+```
+Switch AirconPower                  { channel="gree:airconditioner:a1234561:power" }
+Number AirconMode                   { channel="gree:airconditioner:a1234561:mode" }
+Switch AirconTurbo                  { channel="gree:airconditioner:a1234561:turbo" }
+Switch AirconLight                  { channel="gree:airconditioner:a1234561:light" }
+Number AirconTemp "Temperature [%.1f °C]" {channel="gree:airconditioner:a1234561:temperature" }
+Number AirconTempSet                { channel="gree:airconditioner:a1234561:temperature" }
```

Comment:
Why are there two items for the same channel here?
Also they should be `Number:Temperature`.

Reply:
fixed